lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -98,5 +98,10 @@ public class TopologyRootTaskId : Name<string>
         public class TopologyChildTaskIds : Name<ISet<string>>
         {
         }
+
+        [NamedParameter("Type of the message")]
+        public class MessageType : Name<string>
```

Comment:
I don't understand why we need to convey types as strings. Can you explain the need?